docs/getting-started/tutorials/auth-architecture/enterprise-architecture.md

```diff
@@ -0,0 +1,15 @@
+---
+id: enterprise-architecture 
+title: Enterprise Architecture with edge gateways
+sidebar_label: Enterprise Architecture
+---
+
+## Definition
+
+Entrprise Architecture with edge gateways
+- Use edge gateway(s) to centralize user resolution, API Keys, Ingresses (differentiate Ingresses exposed on the internet vs internal)
```

Comment:
Beware, if the API Key is present between the headers of a request, the API Gateway will always perform a check. So you cannot centralize the API Keys on the Edge Gateway without duplicating the API Keys on the other API Gateways.

Reply:
true, but you could use a small trick. Your projects (except for the edge gateway) accept all the same API Key and you add a behaviour on the Edge Gateway that forwards to the downstream projects the API key accepted by all of your projects.